backend/tests/integration/test_service_integration.py

```
@@ -0,0 +1,512 @@
+import pytest
+import asyncio
+from unittest.mock import Mock, AsyncMock, patch
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)